Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY ./config.yaml /app/config.yaml
 
 VOLUME /data
 
-RUN pip install --no-cache-dir -r /app/requirements.txt
+RUN pip install --no-cache-dir -r ./requirements.txt
```

Comment:
It should be /requirements.txt - same with COPY.

```
COPY ./requirements.txt /requirements.txt
```

And:

```
RUN pip install --no-cache-dir -r /requirements.txt
```

Reply:
Yes, you are right. I tried that yesterday but kept getting errors. I must have been doing something wrong. This is much harder than I expected. I will make that change. Thanks again.